src/_igraph/graphobject.c

```diff
@@ -12902,6 +12981,39 @@ struct PyMethodDef igraphmodule_Graph_methods[] = {
    "  multiplying by the number of vertices minus one.\n"
    "@return: the calculated closenesses in a list\n"},
 
+  /* interface to igraph_harmonic_centrality */
+  {"harmonic_centrality", (PyCFunction) igraphmodule_Graph_harmonic_centrality,
+   METH_VARARGS | METH_KEYWORDS,
+   "harmonic_centrality(vertices=None, mode=ALL, cutoff=None, weights=None,\n"
+   "          normalized=True)\n\n"
+   "Calculates the harmonic centralities of given vertices in a graph.\n\n"
+   "The harmonic centerality of a vertex measures how easily other\n"
+   "vertices can be reached from it (or the other way: how easily it\n"
+   "can be reached from the other vertices). It is defined as the\n"
+   "number of vertices minus one times the sum of one divided by \n"
+   "the length of each geodesic from/to the given vertex.\n\n"
+   "If the graph is not connected, and there is no path between two\n"
+   "vertices, the number of vertices is used instead the length of\n"
+   "the geodesic. This is always longer than the longest possible\n"
+   "geodesic.\n\n"
+   "@param vertices: the vertices for which the harmonic centrality must\n"
+   "  be returned. If C{None}, uses all of the vertices in the graph.\n"
+   "@param mode: must be one of L{IN}, L{OUT} and L{ALL}. L{IN} means\n"
+   "  that the length of the incoming paths, L{OUT} means that the\n"
+   "  length of the outgoing paths must be calculated. L{ALL} means\n"
+   "  that both of them must be calculated.\n"
+   "@param cutoff: if it is an integer, only paths less than or equal to this\n"
+   "  length are considered, effectively resulting in an estimation of the\n"
```

Comment:
Please do not suggest that the purpose of the range-limited version is estimation. I would avoid the word 'estimation' here.

Reply:
sure, that was the old closeness text so I just removed it now

Reply:
We need to fix up the description of closeness in the python-igraph docs, as we also changed how closeness works recently.